src/Npgsql/Replication/PgOutput/PgOutputReplicationOptions.cs

```diff
@@ -61,19 +64,30 @@ public PgOutputReplicationOptions(IEnumerable<string> publicationNames, ulong pr
         /// <remarks>
         /// This works in PostgreSQL versions 14+
         /// </remarks>
+        // See: https://github.com/postgres/postgres/commit/9de77b5453130242654ff0b30a551c9c862ed661
         public bool? Binary { get; }
 
         /// <summary>
-        /// Enable streaming output
+        /// Enable streaming of in-progress transactions
```

Comment:
OK, so now there's the old "streaming replication", and the new "really streaming streaming replication" :rofl:

Reply:
Yeah.
While in V1 transactions were always streamed as a whole, in V2 they can be broken up into smaller units.